### ISSUE TITLE
 ios version incorrectly replaces default name of Þingvellir with russian description 

### ISSUE THREAD

**Issue by Hjart** [mapsme/omim#8905]
The ios version (apparently the Android version doesn't do this) today for the third time incorrectly replaced the default name of the Icelandic [Þingvellir](https://www.openstreetmap.org/node/316032856/history) with the same russian description. Looks like Maps.me stores the russian description somewhere and replaces the default name whenever someone does other edits to the POI.

**Comment by Alexuser1** [comment]
Hello @Hjart !

Please indicate - what did you do before you received the specified situation.

Thanks,
Alexander

**Comment by Hjart** [comment]
I didn't do anything. I noticed [this changeset](https://www.openstreetmap.org/changeset/60000334) , wondered why the name of Þingvellir was replaced, then reverted the change. Then I checked the history of the node and noticed that exactly the same thing had happened on 2 other occasions 1½ - 2 years ago. The ios version of Maps.me appears to be doing something very weird here.

**Comment by Alexuser1** [comment]
Hello @Hjart !

It's usual POI with"attraction" tag. So any users can edit it.

Thanks,
Alexander

**Comment by Hjart** [comment]
> It's usual POI with"attraction" tag. So any users can edit it.

Yes, I'm aware that any users can edit it. I've been editing OSM myself for 9 years. But in this case; Are you trying to tell me that 3 different newbie Maps.me users on 3 different occassions several years apart would replace the default name with exactly the same russian text? Only 1 of these users appears to have been a russian.

I have also been looking at and correcting/reverting lots of Maps.me edits over the years and never before seen anything like this.
